tables.sql

```diff
@@ -0,0 +1,87 @@
+CREATE DATABASE thebackups_TaylorC;
+
+USE thebackups_TaylorC;
+
+CREATE TABLE employee_type (
+	`employee_type_id` TINYINT UNSIGNED AUTO_INCREMENT PRIMARY KEY,
+    `name` VARCHAR(30) NOT NULL
+);
+
+CREATE TABLE employee (
+	`employee_id` SMALLINT UNSIGNED AUTO_INCREMENT PRIMARY KEY,
+    `first_name` VARCHAR(35) NOT NULL,
+    `last_name` VARCHAR(35) NOT NULL,
+    `salary` MEDIUMINT UNSIGNED NOT NULL,
+    `sort_code` VARCHAR(6) NOT NULL,
+    `account_number` VARCHAR(8) NOT NULL,
+    `nin` VARCHAR(9) NOT NULL,
```

Comment:
This could be confusing for what `nin` stands for

Reply:
All Fixed!